generators/app/index.js

```diff
@@ -43,6 +43,15 @@ module.exports = class extends Generator {
         message: 'What\'s the name of your project?',
         default: 'Toolbox'
       }, {
+        type: 'input',
+        name: 'remote',
+        message: 'What\'s the URL of your repo?',
+        default: function (answers) {
```

Comment:
In the `default` below? That's the default value, you can change it in the prompt